applications/launchpad/gui-react/src/containers/Dashboard/ExpertView/Containers/index.tsx

```diff
@@ -23,10 +27,32 @@ const ContainersContainer = () => {
     [containerStatuses],
   )
 
-  const start = (container: Container) => dispatch(actions.start(container))
-  const stop = (containerId: ContainerId) => dispatch(actions.stop(containerId))
+  const start = async (container: Container) => {
+    try {
+      await dispatch(actions.start(container)).unwrap()
+    } catch (e: any) {
+      setError(e.toString())
+    }
+  }
```

Comment:
Idea: Let's say that Tor is not running and the user starts the base node. Maybe in such case the app can check if all dependencies are running. If not, prompt the alert:

```
Base node requires following containers to be running: x, y, z.
Do you want to run also dependent containers?

[NO/Cancel]                 [Yes/Accept]
```

Reply:
that's interesting!
we could also have TBot say those suggestions, like the OG Clippy :D

(but let's not add ourselves work right now, this could be a future feature)